### PR TITLE
[languageserver] Update to Cadence v1.0.0-preview.25

### PR DESCRIPTION
**Pull request by turbolent**
## Description

Automatically update to:
- [onflow/cadence v1.0.0-preview.25](https://github.com/onflow/cadence/releases/tag/v1.0.0-preview.25)
- [onflow/flow-go-sdk v1.0.0-preview.25](https://github.com/onflow/flow-go-sdk/releases/tag/v1.0.0-preview.25)
- [onflow/flow-go v0.34.0-crescendo-preview.18](https://github.com/onflow/flow-go/releases/tag/v0.34.0-crescendo-preview.18)
- [onflow/flow-emulator v1.0.0-preview.22](https://github.com/onflow/flow-emulator/releases/tag/v1.0.0-preview.22)
- [onflow/cadence-tools/lint v1.0.0-preview.16](https://github.com/onflow/cadence-tools/lint/releases/tag/v1.0.0-preview.16)
- [onflow/cadence-tools/test v1.0.0-preview.15](https://github.com/onflow/cadence-tools/test/releases/tag/v1.0.0-preview.15)
- [onflow/flowkit/v2 v2.0.0-stable-cadence-alpha.17](https://github.com/onflow/flowkit/v2/releases/tag/v2.0.0-stable-cadence-alpha.17)

### CODE REVIEW

File: languageserver/go.mod [hunk 1]
```diff
@@ -1,17 +1,19 @@
 module github.com/onflow/cadence-tools/languageserver
 
-go 1.18
+go 1.21
+
+toolchain go1.21.9
```
Comment (turbolent) [inline languageserver/go.mod:5]: Looks like flowkit is already on Go 1.21 now

**Review by SupunS — APPROVED** [review]

**Comment by turbolent** [comment]
Guess because of the Go version bump to 1.21 we need to update `npm-packages/cadence-language-server/src/go.js` again 🙄

**Comment by turbolent** [comment]
No idea why the Go 1.21 breaks the WASM LS. From 1.20 to 1.21, something seems to have changed about timeouts. I tried a few things but no luck :/

**Comment by turbolent** [comment]
All tests actually pass. What causes the CI failure is that after all tests have run, the test suite's shutdown function notifies the LS to exit, which calls `os.Exit`. Somehow this causes a timing issue starting in Go 1.21.

Will merge and we'll need to address this in a follow-up.

**Pull request merged**